src/main/java/com/example/hubtjobsreactive/config/SecurityConfig.java

```diff
@@ -25,8 +25,10 @@ SecurityWebFilterChain springSecurityFilterChain(ServerHttpSecurity http) {
                 .authorizeExchange()
                 .pathMatchers("/poster/**")
                 .hasAnyRole("poster")
-                .pathMatchers("/**")
-                .authenticated()
+//                .pathMatchers("/**")
```

Comment:
remove commented code if you're not planning to use it in the future

Reply:
I think we should change to pathMatchers("/actuator/**").permitAll().anyExchange().authenticated()